app/common/gristUrls.ts

```diff
@@ -891,6 +894,15 @@ export function getFreeCoachingCallUrl(): string|null {
   }
 }
 
+export function contactSupportUrl(): string|null {
+  if(isClient()) {
+    const gristConfig: GristLoadConfig = (window as any).gristConfig;
+    return gristConfig && gristConfig.contactSupportUrl || null;
+  } else {
+    return process.env.CONTACT_SUPPORT_URL || null;
+  }
+}
+
```

Comment:
This and `getFreeCoachingCallUrl` are both unused, right? We've used functions like these in the past in `gristUrls.ts` to unify shared URLs in client and server code. That's probably not a bad idea here, as we reach for URLs from that file often, and we may not always remember to use the URLs from `window.gristConfig`.

It should be an easy refactor if you're up for it: updating `commonUrls` to call these functions for the `contact` and (new) `freeCoachingCall` keys, and updating `errorPages.ts` (and wherever the coaching call URL currently lives) to reference them. I'd also change the `|| null` to `|| <fallback url>`, and update `makeGristConfig` in `sendAppPage.ts` to replace the URLs like `process.env.CONTACT_SUPPORT_URL || "https://www.getgrist.com/contact/"` with calls to these functions as well. Wdyt?